samples/rag-cosmosdb/README.md

```diff
@@ -22,39 +22,53 @@ Once you have an Cosmos DB resource, you can run the sample by following these s
 1. Always configure the search provider type in the `host.json` as shown in below snippet.
 1. Use of Vector Search Dimensions and the number of clusers that the inverted file (IVF) index uses to group the vector data are configurable. You may configure the `host.json` file within the project and following example shows the default values:
 
-    ```json
-    "extensions": {
-        "openai": {
-            "searchProvider": {
-                "type": "cosmosDBSearch",
-                "vectorSearchDimensions": 1536,
-                "numLists":  1
-            }
-        }
-    }
-    ```
-
-    `VectorSearchDimensions` is length of the embedding vector. [The dimensions attribute has a minimum of 2 and a maximum of 2000 floating point values each](https://learn.microsoft.com/azure/cosmos-db/mongodb/vcore/vector-search#create-an-vector-index-using-ivf). By default, the length of the embedding vector will be 1536 for text-embedding-ada-002.
-
-    `NumLists` is the number of clusters that the inverted file (IVF) uses to group the vector data, as mentioned [here](https://learn.microsoft.com/azure/cosmos-db/mongodb/vcore/vector-search#create-an-vector-index-using-ivf). By default, the number of clusters will be 1.
+   ```json
+   "extensions": {
+       "openai": {
+           "searchProvider": {
+               "type": "cosmosDBSearch",
+               "connectionString": "",
+               "databaseName": "functionsDatabase",
+               "collectionName": "functionsCollection",
+               "indexName": "functionsIndex",
+               "applicationName": "functionsAppName",
+               "embeddingKey": "embedding",
+               "vectorSearchDimensions": 1536,
+               "numLists":  1,
+               "kind": "vector-hnsw",
+               "similarity": "COS",
+               "numberOfConnections": 16,
+               "efConstruction": 64,
+               "efSearch": 40
+     }
+   }
+           }
+       }
+   }
+   ```
+
+   `VectorSearchDimensions` is length of the embedding vector. [The dimensions attribute has a minimum of 2 and a maximum of 2000 floating point values each](https://learn.microsoft.com/azure/cosmos-db/mongodb/vcore/vector-search#create-an-vector-index-using-ivf). By default, the length of the embedding vector will be 1536 for text-embedding-ada-002.
+
+   `NumLists` is the number of clusters that the inverted file (IVF) uses to group the vector data, as mentioned [here](https://learn.microsoft.com/azure/cosmos-db/mongodb/vcore/vector-search#create-an-vector-index-using-ivf). By default, the number of clusters will be 1.
 
```

Comment:
new settings should also be explained here and default values for them if unspecified